components/eventing-controller/pkg/env/nats_config_test.go

```diff
@@ -16,12 +18,14 @@ func Test_GetNatsConfig(t *testing.T) {
 	idleConnTimeout := time.Second * 40
 
 	envs := map[string]string{
-		"NATS_URL":                "NATS_URL",
-		"EVENT_TYPE_PREFIX":       "EVENT_TYPE_PREFIX",
-		"MAX_IDLE_CONNS":          fmt.Sprintf("%d", maxIdleConns),
-		"MAX_CONNS_PER_HOST":      fmt.Sprintf("%d", maxConnsPerHost),
-		"MAX_IDLE_CONNS_PER_HOST": fmt.Sprintf("%d", maxIdleConnsPerHost),
-		"IDLE_CONN_TIMEOUT":       fmt.Sprintf("%v", idleConnTimeout),
+		"NATS_URL":                 "NATS_URL",
+		"JS_STREAM_NAME":           "kyma",
+		"JS_STREAM_SUBJECT_PREFIX": "kyma",
```

Comment:
We should check that those values are reflected in the `config` instance?
We also want to check that empty values are not accepted.

Reply:
I have added a check for these keys. Also the empty case is addressed in the next test [here](https://github.com/kyma-project/kyma/blob/80b9c00a8a732ee502d2967a46d10d0acf539adb/components/eventing-controller/pkg/env/nats_config_test.go#L94)